src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -116,8 +128,8 @@ public void execute_multipleScheduleKeywords_multipleStudentsFound() {
     }
 
     @Test
-    public void execute_nameAndScheduleKeywords_multipleStudentsFound() {
-        String expectedMessage = String.format(MESSAGE_STUDENTS_LISTED_OVERVIEW, 2);
+    public void execute_nameAndScheduleKeywords_noStudentFound() {
+        String expectedMessage = String.format(MESSAGE_STUDENTS_LISTED_OVERVIEW, 0);
```

Comment:
Looks good!
But if so, I think need to specify in the UG also, e.g., give an example to tell them that if both name and day are given, then only search for students containing 1 of the names **and** 1 of the days.

Reply:
Hi, thank you for catching this. documentation updated here. #254